### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by Marklord-dev** [Cyfrin/foundry-full-course-cu#1953] [Q&A]
the code 
function retrieve() public view returns(uint256){
        return favoriteNumber;
    } 

is not giving me the retrieve section after deploying it on remix. maybeb there is something i'm not getting right. Please help
![remix](https://github.com/Cyfrin/foundry-full-course-cu/assets/174838107/8b02e87a-55d9-4dc8-94b0-dcb1cf81ea45)

**Comment by EngrPips** [comment]
Hello @Marklord-dev, delete the contract and re-deploy. If the problem persists, refresh your browser and retry; if it persists, share the code. Let me try it in remix and see how it goes for me.

**Reply by Marklord-dev** [reply]
it worked. Thank you

**Reply by EngrPips** [reply]
Awesome, I am glad it worked. Keep crushing it, buddy.
